src/css/custom.css

```diff
@@ -93,6 +95,24 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: 'Green Brooks';
+  src: url('../../static/fonts/GreenBrooks.woff2') format('woff2'),
+      url('../../static/fonts/GreenBrooks.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Glancyr';
+  src: url('../../static/fonts/Glancyr-Regular.woff2') format('woff2'),
+      url('../../static/fonts/Glancyr-Regular.woff') format('woff');
+  font-weight: 400;
```

Comment:
should cover all supported font weights

Reply:
Zee asked to add only weight 400..and convert all weights to 400 only